zookeeper-client/zookeeper-client-c/tests/TestZookeeperInit.cc

```diff
@@ -90,6 +91,20 @@ class Zookeeper_init : public CPPUNIT_NS::TestFixture
 #endif
     }
 
+    void testVersion()
+    {
+        char EXPECTED_VERSION_STR[256];
+        EXPECTED_VERSION_STR[0] = 0;
+#ifdef ZOO_VERSION
+	snprintf(EXPECTED_VERSION_STR, sizeof(EXPECTED_VERSION_STR), "%s", ZOO_VERSION);
+#endif
+#ifdef ZOO_VERSION_MAJOR
```

Comment:
Since the version macro change is introduced https://github.com/apache/zookeeper/commit/253673f30431344030509b974bd2f65d23c1cd6e in 3.6.0, I don't think we will do backport that needs this branch. That is, we can retain only:

```c
snprintf(EXPECTED_VERSION_STR, sizeof(EXPECTED_VERSION_STR), "%s", ZOO_VERSION);
```